lib/node_modules/@stdlib/stats/incr/nanmsumabs/README.md

```diff
@@ -0,0 +1,179 @@
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var incrnanmsumabs = require( '@stdlib/stats/incr/nanmsumabs' );
+```
+
+#### incrnanmsumabs( window )
+
+Returns an accumulator `function` which incrementally computes a moving sum of absolute values. The `window` parameter defines the number of values over which to compute the moving sum, ignoring `NaN` values.
+
+```javascript
+var accumulator = incrnanmsumabs( 3 );
+```
+
+#### accumulator( \[x] )
+
+If provided an input value `x`, the accumulator function returns an updated sum. If not provided an input value `x`, the accumulator function returns the current sum.
+
+```javascript
+var accumulator = incrnanmsumabs( 3 );
+
+var sum = accumulator();
+// returns null
+
+// Fill the window...
+sum = accumulator( 2.0 ); // [2.0]
+// returns 2.0
+
+sum = accumulator( NaN ); // [2.0]
+// returns 2.0
+
+sum = accumulator( -1.0 ); // [2.0, -1.0]
+// returns 3.0
+
+sum = accumulator( 3.0 ); // [2.0, -1.0, 3.0]
+// returns 6.0
+
+// Window begins sliding...
+sum = accumulator( -7.0 ); // [-1.0, 3.0, -7.0]
+// returns 11.0
+
+sum = accumulator( -5.0 ); // [3.0, -7.0, -5.0]
+// returns 15.0
+
+sum = accumulator();
+// returns 15.0
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   Input values are **not** type checked. If non-numeric inputs are possible, you are advised to type check and handle accordingly **before** passing the value to the accumulator function.
+-   As `W` values are needed to fill the window buffer, the first `W-1` returned values are calculated from smaller sample sizes. Until the window is full, each returned value is calculated from all provided values.
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
+var incrnanmsumabs = require( '@stdlib/stats/incr/nanmsumabs' );
+
+var accumulator;
+var v;
+var i;
+
+// Initialize an accumulator:
+accumulator = incrnanmsumabs( 5 );
+
+// For each simulated datum, update the moving sum...
+for ( i = 0; i < 100; i++ ) {
+    if ( randu() < 0.2 ) {
+        v = NaN;
+    } else {
+        v = ( randu()*100.0 ) - 50.0;
+    }
+    accumulator( v );
+}
+console.log( accumulator() );
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/stats/incr/mmeanabs`][@stdlib/stats/incr/mmeanabs]</span><span class="delimiter">: </span><span class="description">compute a moving arithmetic mean of absolute values incrementally.</span>
```

Comment:
Same comments as other PRs. You'll want to apply similar changes here and elsewhere.